retrodeck.sh

```diff
@@ -185,9 +185,10 @@ standalones_init() {
 
 ra_init() {
     dir_prep "$rdhome/bios" "/var/config/retroarch/system"
-     dir_prep "$rdhome/.logs/retroarch" "/var/config/retroarch/logs"
+    dir_prep "$rdhome/.logs/retroarch" "/var/config/retroarch/logs"
     mkdir -pv /var/config/retroarch/cores/
     cp /app/share/libretro/cores/* /var/config/retroarch/cores/
+    grab_extra_ra_cores
```

Comment:
This function still not exist here, I think you should move the grab_extra_ra_cores() function before of it.